packages/skygear-core/lib/database.js

```diff
@@ -33,6 +33,10 @@ export default class Database {
     this._cacheResponse = true;
   }
 
+  get Record() {
+    return this.container.Record;
+  }
```

Comment:
I am in double the Record are accessible from boath BaseContainer and database container. Semm we should choose one. In this case, I will choose BaseContainer.